### ISSUE TITLE
plugins set as "global" are not modifiable

### ISSUE THREAD

**Issue by ghost** [liyuntao/kong-init#7]
plugins set as "global" are not modifiable.

create one...
```
plugins:
  - name: rate-limiting
    target: global
    enabled: true
    config:
      minute: 101
```

now go run the `kong-init` binary and then `httpie  http://localhost:8001/plugins`

then 
modify it...
```
plugins:
  - name: rate-limiting
    target: global
    enabled: false <====== changed this
    config:
      minute: 101
```

now go run the `kong-init` binary and then `httpie  http://localhost:8001/plugins`

you will see that it is still `enabled: true` in Kong.

I cannot delete them too

Kong 0.14.1, postgres

**Comment by liyuntao** [comment]
Thanks for the detailed report here!  It seems that there is a logical missing here(clear all existed global plugin before init action). 

Will fix it soon.

**Comment by liyuntao** [comment]
Just pre-released a version `v0.8.0-beta-3`. It will clear all plugins(not only global plugin) before doing initialization. 

Any feedback is welcome.

**Comment by ghost** [comment]
it works thanks for fix 👍
